docs/static/releasenotes.asciidoc

```diff
@@ -1,12 +1,55 @@
 [[releasenotes]]
 == Release Notes
 
-This section summarizes the changes in 5.0.0 release.
+This section summarizes the changes in the following releases:
+
+* <<logstash-5-0-1,Logstash 5.0.1>>
+* <<logstash-5-0-0,Logstash 5.0.0>>
 
 List of <<breaking-changes, breaking changes>>
 
+[[logstash-5-0-1]]
+=== Logstash 5.0.1 Release Notes
+
+* Logstash can now start without logstash.yml, even though this is not ideal and not recommended for production. 
+  Requiring `logstash.yml` in 5.0.0 caused confusion among existing users who install LS from a package and tried to start it 
+  manually via `bin/logstash` ({lsissue}6170[Issue 6170]).
+* Fixed a bug when incoming data already had a field called `tags` and Logstash errored trying to append more 
+  tags to it ({lsissue}6142[Issue 6142]).
+* Increased the current polling interval of a background thread that retrieves stats from JVM and OS layer from 1s to 5s ({lsissue}6160[Issue 6160]).
+* Merge config strings when both `-e` and `-f` was supplied even if `-f` does not yield in any config files ({lsissue}6164[Issue 6164]).
+* Improve user feedback message when starting Logstash manually after a package install ({lsissue}6061[Issue 6061]).
+
 [float]
-==== General
+==== Input Plugins
+
+*`Beats`*:
+
+* The plugin can now bind to a specific host using the `host` config (https://github.com/logstash-plugins/logstash-input-beats/issues/146[Issue 146]).
+* Fixed an issue where only the first certificate found in the CA chain were taken into consideration to verify clients (https://github.com/logstash-plugins/logstash-input-beats/issues/153[Issue 153]).
```

Comment:
Change to: "...only the first certificate found in the CA chain was taken into consideration..."